### PR TITLE
Support floc origin-trial

### PR DESCRIPTION
**Pull request by jlukas79**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: test/spec/modules/eids_spec.js [hunk 1]
```diff
@@ -304,15 +304,18 @@ describe('eids array generation for known sub-modules', function() {
   });
   it('originFloc', function() {
     const userId = {
-      flocId: {'id': 'Sample_Token'}
+      flocId: {id: 'Sample_Token', ver: 'chrome1.1'}
```
Comment (jdwieland8282) [inline test/spec/modules/eids_spec.js:307]: is "sample_token" the origin trial token from the meta tag or the floc cohort id.  It should be the cohort id, and should probably have a name w/o token in it.

File: modules/flocIdSystem.js [hunk 2]
```diff
@@ -43,21 +37,41 @@ export const flocIdSubmodule = {
...
    */
   getId(config) {
-    const configParams = (config && config.params) || {};
-    if (!configParams || (typeof configParams.token !== 'string')) {
-      utils.logError('User ID - flocId submodule requires token to be defined');
-      return;
+    const isChrome = !!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime);
+
+    // Validate feature is enabled
+    if (isChrome && !!document.featurePolicy && !!document.featurePolicy.features() && document.featurePolicy.features().includes('interest-cohort')) {
```
Comment (iamnewton) [inline modules/flocIdSystem.js]:
```suggestion
    const isFlocEnabled = !!document.featurePolicy && !!document.featurePolicy.features() && document.featurePolicy.features().includes('interest-cohort'));

    if (isChrome && isFlocEnabled) {
```

File: modules/flocIdSystem.js [hunk 3]
```diff
@@ -43,21 +37,41 @@ export const flocIdSubmodule = {
...
+      if (checkStorage) {
+        utils.logError('User ID - flocId submodule requires storage not defined');
+        return;
+      }
+
+      // Insert meta-tag with token from configuration
+      enableOriginTrial(configParams.token);
+
+      // Example expected output { "id": "14159", "version": "chrome.1.0" }
+      const cohort = document.interestCohort() || {};
```
Comment (iamnewton) [inline modules/flocIdSystem.js]:
```suggestion
      const {id, version as ver } = document.interestCohort() || {};
```

File: modules/flocIdSystem.js [hunk 4]
```diff
@@ -43,21 +37,41 @@ export const flocIdSubmodule = {
...
+
+      return { id: cohort.id, ver: cohort.version };
```
Comment (iamnewton) [inline modules/flocIdSystem.js]:
```suggestion
      return { id, ver };
```

File: modules/flocIdSystem.js [hunk 5]
```diff
@@ -43,21 +37,41 @@ export const flocIdSubmodule = {
    * @returns {{flocId:{ id: string }} or undefined if value doesn't exists
    */
   decode(value) {
-    return (value) ? encodeId(value) : undefined;
+    return undefined;
   },
```
Comment (iamnewton) [inline modules/flocIdSystem.js]: just delete the entire function rather than return nothing

**Pull request merged**